lib/krikri/harvesters/couchdb_harvester.rb

```diff
@@ -20,6 +20,7 @@ def initialize(opts = {})
       super
       @opts = opts.fetch(:couchdb, view: '_all_docs')
       @opts[:view] ||= '_all_docs'
+      @opts[:limit] ||= 10
```

Comment:
IMO, this should match whatever CouchDB's external interface provides, so I think `:limit` is correct.